lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1824,6 +1824,12 @@ sub test_resultfile_list {
         }
     }
 
+    for my $i (1 .. 20) {
```

Comment:
if you do not use a magic number here I agree to merge it despite being duplication. We still have the ticket to incorporate the suggestions that were mentioned in comments.